azure-functions/src/durable.rs

```diff
@@ -12,6 +12,7 @@ use std::{
     task::{Context, Poll, RawWaker, RawWakerVTable, Waker},
 };
 
+mod activity_output;
```

Comment:
We should export everything in that module into the current (`durable`) module with `pub use self::activity_output::*;` (add to the list of below `pub use` statements).